src/main/java/com/basho/riak/client/core/query/indexes/RiakIndex.java

```diff
@@ -352,7 +352,7 @@ public String toString()
      * @param <T> the RiakIndex this Name encapsulates
      * @see RiakIndexes
      */
-    public static abstract class Name<T extends RiakIndex>
+    public static abstract class Name<T extends RiakIndex<?>>
```

Comment:
So this basically says that `RiakIndex` will be a generic type without any type constraints?